data.sql

```diff
@@ -1,5 +1,9 @@
 /* Populate database with sample data. */
 
```

Comment:
- [ ] kindly consider adding a transaction:to UPDATE the animal's table by setting the species column to unspecified, then roll back the change ( make sure that they included all steps).
hint: 
```
   BEGIN; -- start transaction

   UPDATE animals
   SET species = 'unspecified'; -- make change

   SELECT species from animals; -- verify that change was made
   ROLLBACK;

   SELECT species from animals; -- verify that change was undone

```
to meet the below project requirement 👇 
<img width="1068" alt="Screen Shot 2023-08-17 at 08 52 26" src="https://github.com/yemidada/VetClinicDatabase/assets/60146030/d124cee6-22d0-4ab5-ac53-eb1aa47d52e4">